SYCL/ESIMD/api/functional/type_coverage.hpp

```diff
@@ -248,6 +248,8 @@ template <tested_types required> auto get_tested_types() {
                                       "unsigned int", "long", "unsigned long",
                                       "float", "sycl::half", "double",
                                       "long long", "unsigned long long");
+  } else if constexpr (required == tested_types::small) {
+    return named_type_pack<float, int>::generate("float", "int");
```

Comment:
```suggestion
    return named_type_pack<float, int, unsigned int, signed char>::generate("float", "int", "unsigned int" , "signed char");
```
By doing so we would have the following classes covered:

- unsigned integral types by `unsigned int`
- signed integral types of different sizes by `signed char` and `int`
- floating point types by `float`